### ISSUE TITLE
[Legitimate Site Blocked] e995320s.beget.tech

### ISSUE THREAD

**Issue by LNDDuslys** [MetaMask/eth-phishing-detect#8578]
http://e995320s.beget.tech/

**Comment by AlexHerman1** [comment]
not sure what the nature of this site is, if you can provide more context and why its an issue that its blocked by metamask please let us know in a new PR.
